great_expectations/agent/actions/test_draft_datasource_config.py

```diff
@@ -0,0 +1,49 @@
+from uuid import UUID
+
+import pydantic
+
+from great_expectations.agent.actions import ActionResult, AgentAction
+from great_expectations.agent.agent import GxAgentEnvVars
+from great_expectations.agent.models import DraftDatasourceConfig
+from great_expectations.core.http import create_session
+
+
+class DraftDatasourceConfigAction(AgentAction[DraftDatasourceConfig]):
+    def run(self, event: DraftDatasourceConfig, id: str) -> ActionResult:
```

Comment:
See other comment on naming confusion.

Reply:
thanks, updated this as well - not sure what i was thinking when i named these 🙃